aptos-move/framework/aptos-framework/sources/fungible_asset.move

```diff
@@ -307,8 +328,12 @@ module aptos_framework::fungible_asset {
 
     #[view]
     /// Get the balance of a given store.
-    public fun balance<T: key>(store: Object<T>): u64 acquires FungibleStore {
-        if (store_exists(object::object_address(&store))) {
+    public fun balance<T: key>(store: Object<T>): u64 acquires FungibleStore, ConcurrentFungibleBalance {
+        let store_addr = object::object_address(&store);
+        if (concurrent_fungible_balance_exists(store_addr)) {
+            let balance = borrow_global<ConcurrentFungibleBalance>(store_addr);
```

Comment:
I usually prefer
```
let balance = &borrow_global<ConcurrentFungibleBalance>(store_addr).balance;
aggregator_v2::read(balance)
```
because `balance.balance` looks super weird. Either things need to be called differently, or we can access the field directly when we borrow global